script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
+    with open('scripts/example_data.yaml') as yaml_file:
```

Comment:
Similarly, use 'yaml.load' directly with 'with open' to simplify the YAML file handling.
```suggestion
        print(yaml.dump(yaml.load(yaml_file, Loader=yaml.SafeLoader), indent=2))
```